packages/core/src/layout/HeaderTabs/index.tsx

```diff
@@ -78,7 +78,7 @@ export const HeaderTabs: React.FC<{
         {tabs.map((tab, index) => (
           <Tab
             label={tab.label}
-            key={tab.id}
+            key={`${tab.id}_${index}`}
```

Comment:
This doesn't seem correct, would expect the user of the component needs to be changed instead? If it's a fix for a warning it'd be good to submit this as a separate PR!